package.json

```diff
@@ -33,7 +33,7 @@
   "homepage": "https://github.com/kellyselden/ember-cli-update#readme",
   "dependencies": {
     "debug": "^3.0.0",
-    "ember-modules-codemod": "^0.2.9",
+    "ember-modules-codemod": "kellyselden/ember-modules-codemod#mixed-eol",
```

Comment:
This was published I think (0.2.11 maybe?)...

Reply:
updated.